docs/source/FPGA-to-bitstream/Bitstream generation.rst

```diff
@@ -46,11 +46,11 @@ With the FASM file generated by Nextpnr or VPR, you can run this command for bit
 Example
 ^^^^^^^
 
-The following are simple commands to synthesis the netlist ``sequential_16bit_en`` into a JSON netlist.
+The following are simple commands to synthesize the netlist ``sequential_16bit_en`` into a bitstream file.
```

Comment:
I think maybe synthesize is a misleading term, as we'd normally associate that with synthesis into a netlist. How about:
```suggestion
The following command lowers the FASM representation of the placed and routed ``sequential_16bit_en`` design into a bitstream file.
```

Reply:
Yes your're right, the original sentence doesn't make any sense there, but I only saw the typo. Thanks!